theme/brb/views/imprint/imprint.hbs

```diff
@@ -26,93 +29,4 @@
 		Die Schul-Cloud Brandenburg ist ein nicht kommerzielles Pilotprojekt, welches vom HPI in Kooperation mit dem Ministerium für Bildung, Jugend und Sport des Landes Brandenburg (<a href="https://mbjs.brandenburg.de" target="_blank" rel="noopener">https://mbjs.brandenburg.de</a>) und der Digital Agentur Brandenburg GmbH (<a href="https://www.ilb.de/de/wir-ueber-uns/beteiligungen/weitere-beteiligungen/digitalagentur-brandenburg-gmbh/" target="_blank" rel="noopener">https://www.ilb.de/de/wir-ueber-uns/beteiligungen/weitere-beteiligungen/digitalagentur-brandenburg-gmbh/</a>) durchgeführt wird. Die Inhalte der Schul-Cloud Brandenburg stellen keine offiziellen Verlautbarungen des HPI dar. Für die Inhalte sind die jeweiligen Autor/innen selbst verantwortlich. Ohne vorherige schriftliche Genehmigung ist eine kommerzielle Verbreitung der über die Schul-Cloud Brandenburg verfügbaren Inhalte ausdrücklich untersagt, außer diese sind ausdrücklich gekennzeichnet. Die redaktionelle Verantwortung für den Inhalt verlinkter Webseiten anderer Anbieter (einschließlich der dort verankerten Links zu Dritten), insbesondere die Verantwortung für die Beachtung der Bestimmungen des Jugendschutz- und Datenschutzrechts, trägt in jeder Hinsicht der jeweilige Anbieter allein.</p>
```

Comment:
replace Schul-Cloud Brandenburg with theme name

Reply:
Why? It's in the specific theme directory

Reply:
Maybe it's a good idea after all. If we ever need to rebrand the Brandenburg Cloud, it would be cool if the variable is used.

Reply:
All new main files contain theme variables so if the platform name is the only text difference compared to main, there is no need for a separate theme file, just remove it and theme-specific text will be used. If I use a separate file here, there is a text difference in it besides platform title.